eZ/Publish/Core/Repository/ContentTypeService.php

```diff
@@ -1639,4 +1640,20 @@ public function removeContentTypeTranslation(APIContentTypeDraft $contentTypeDra
 
         return $this->contentTypeDomainMapper->buildContentTypeDraftDomainObject($contentType);
     }
+
+    public function deleteUserDrafts(int $userId): void
+    {
+        try {
+            $this->repository->getUserService()->loadUser($userId);
```

Comment:
Please inject `eZ\Publish\SPI\Persistence\User\Handler as UserHandler` and use it here due to performance and future potential cyclic dependency issues.